### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by kdomino**
new notation compatible with the work of thers

### CODE REVIEW

File: src/runtests.jl [hunk 1]
```diff
@@ -1,28 +1,44 @@
-using Test
 using LinearAlgebra
```
Comment (lpawela) [inline src/runtests.jl]: What is the purpose of this entire file?

File: src/tests/data/chimera2048_spinglass_power/error_note.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+kdomino@squeak:~/Dokumenty/badania/peps/src/tests$ julia tests_on_chimera.jl -c 5 --file data/chimera2048_spinglass_power/001.txt -s 2048
```
Comment (lpawela) [inline src/tests/data/chimera2048_spinglass_power/error_note.txt]: Why is this here???

File: src/tests/mps_tests.jl [hunk 3]
```diff
@@ -1,198 +1,85 @@
 using TensorOperations
```
Comment (lpawela) [inline src/tests/mps_tests.jl]: Test files should be in `tests` not `src/tests`

File: test/runtests.jl [hunk 4]
```diff
@@ -22,11 +22,12 @@ if CUDA.functional() && CUDA.has_cutensor()
 end
 
 push!(my_tests,
-    "base.jl",
-   "contractions.jl",
-   "compressions.jl",
-   "ising.jl",
-   "search.jl"
+#    "base.jl",
```
Comment (lpawela) [inline test/runtests.jl]: All tests should be uncommented before merging

**Review by lpawela — CHANGES_REQUESTED** [review]

**Comment by lpawela** [comment]
Some general remarks:
1. Tests should be in `tests` not `src/tests`
2. Why do we need so many instances in testing?
3. NO BINARY OUTPUT FILES IN THE REPO!!! PLEASE REMOVE THEM WITH AN APPROPRIATE HISTORY REWRITE!!!

File: src/runtests.jl (hunk 1 above)
Comment (kdomino) [reply src/runtests.jl]: removed

File: src/tests/data/chimera2048_spinglass_power/error_note.txt (hunk 2 above)
Comment (kdomino) [reply src/tests/data/chimera2048_spinglass_power/error_note.txt]: removed

File: src/tests/mps_tests.jl (hunk 3 above)
Comment (kdomino) [reply src/tests/mps_tests.jl]: done

File: test/runtests.jl (hunk 4 above)
Comment (kdomino) [reply test/runtests.jl]: uncommented

**Review by lpawela — APPROVED** [review]
